op-acceptance-tests/tests/depreqres/depreqres_test.go

```diff
@@ -0,0 +1,56 @@
+package depreqres
+
+import (
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/dsl"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/ethereum-optimism/optimism/op-supervisor/supervisor/types"
+)
+
+func TestUnsafeChainStalling_DisabledReqRespSync(gt *testing.T) {
+	t := devtest.SerialT(gt)
+	sys := presets.NewSingleChainMultiNode(t)
+	require := t.Require()
+	l := t.Logger()
+
+	l.Info("Confirm that the CL nodes are progressing the unsafe chain")
+	target := uint64(10)
+	dsl.CheckAll(t,
+		sys.L2CL.AdvancedFn(types.LocalUnsafe, target, 30),
+		sys.L2CLB.AdvancedFn(types.LocalUnsafe, target, 30),
+	)
+
+	l.Info("Stop the L2 batcher")
+	sys.L2Batcher.Stop()
+
+	l.Info("Disconnect L2CL from L2CLB, and vice versa")
+	sys.L2CLB.DisconnectPeer(sys.L2CL)
+	sys.L2CL.DisconnectPeer(sys.L2CLB)
+
+	ssA_before := sys.L2CL.SyncStatus()
+	ssB_before := sys.L2CLB.SyncStatus()
+
+	l.Info("L2CL status before delay", "unsafeL2", ssA_before.UnsafeL2.ID(), "safeL2", ssA_before.SafeL2.ID())
+	l.Info("L2CLB status before delay", "unsafeL2", ssB_before.UnsafeL2.ID(), "safeL2", ssB_before.SafeL2.ID())
+
+	time.Sleep(20 * time.Second)
+
+	ssA_after := sys.L2CL.SyncStatus()
+	ssB_after := sys.L2CLB.SyncStatus()
+
+	l.Info("L2CL status after delay", "unsafeL2", ssA_after.UnsafeL2.ID(), "safeL2", ssA_after.SafeL2.ID())
+	l.Info("L2CLB status after delay", "unsafeL2", ssB_after.UnsafeL2.ID(), "safeL2", ssB_after.SafeL2.ID())
+
+	require.Greater(ssA_after.UnsafeL2.Number, ssA_before.UnsafeL2.Number, "unsafe chain for L2CL should have advanced")
+	require.Equal(ssB_after.UnsafeL2.Number, ssB_before.UnsafeL2.Number, "unsafe chain for L2CLB should have stalled")
+
+	l.Info("Re-connect L2CL to L2CLB")
+	sys.L2CLB.ConnectPeer(sys.L2CL)
+	sys.L2CL.ConnectPeer(sys.L2CLB)
+
+	l.Info("Confirm that the unsafe chain for L2CLB is stalled, since L2 batcher is stopped, even though ELSync is enabled on L2CLB")
```

Comment:
> even though ELSync is enabled on L2CLB

So we are running the verifier EL with `sync.ELSync`. This means we are enabling the `initial` EL Sync which is for bootstrapping. 

Since the L2CLB received payload for block number 1, L1CLB advanced unsafe head from 0 to 1, since payload 1 can be directly appended to genesis 0. This means the `initial` EL Sync is done.

So the stall is not related with `even though ELSync is enabled on L2CLB` at the logs. 

```suggestion
	l.Info("Confirm that the unsafe chain for L2CLB is stalled")
```